flink-connectors/flink-connector-kafka/pom.xml

```diff
@@ -68,6 +67,20 @@ under the License.
 			<optional>true</optional>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-runtime</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-planner_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+
```

Comment:
Those also should be `optional` (we don't want to force `DataStream API` users to depend on `flink-table-xxx` packages)

Reply:
ah i understand, added optional tag